packages/dd-trace/test/appsec/iast/vulnerability-reporter.spec.js

```diff
@@ -380,6 +380,43 @@ describe('vulnerability-reporter', () => {
     })
   })
 
+  describe('sendVulnerabilities appsec standalone', () => {
+    let span
+
+    beforeEach(() => {
+      span = {
+        addTags: sinon.stub()
+      }
+      start({
+        iast: {
+          deduplicationEnabled: true
+        },
+        appsec: {
+          standalone: {
+            enabled: true
+          }
+        }
+      })
+    })
+    afterEach(() => {
+      sinon.restore()
+      stop()
+    })
+
+    it('should add _dd.p.appsec span tag', () => {
+      const iastContext = { rootSpan: span }
+      addVulnerability(iastContext,
+        vulnerabilityAnalyzer._createVulnerability('INSECURE_HASHING', { value: 'sha1' }, 999))
+      sendVulnerabilities(iastContext.vulnerabilities, span)
+      expect(span.addTags).to.have.been.calledOnceWithExactly({
+        'manual.keep': 'true',
+        '_dd.p.appsec': 1,
+        '_dd.iast.json': '{"sources":[],"vulnerabilities":[{"type":"INSECURE_HASHING","hash":3254801297,' +
+          '"evidence":{"value":"sha1"},"location":{"spanId":999}}]}'
+      })
+    })
```

Comment:
Some spaces, 1 between preparation and method execution, and other between method execution and checks.

```suggestion
    it('should add _dd.p.appsec span tag', () => {
      const iastContext = { rootSpan: span }
      addVulnerability(iastContext,
        vulnerabilityAnalyzer._createVulnerability('INSECURE_HASHING', { value: 'sha1' }, 999))

      sendVulnerabilities(iastContext.vulnerabilities, span)

      expect(span.addTags).to.have.been.calledOnceWithExactly({
        'manual.keep': 'true',
        '_dd.p.appsec': 1,
        '_dd.iast.json': '{"sources":[],"vulnerabilities":[{"type":"INSECURE_HASHING","hash":3254801297,' +
          '"evidence":{"value":"sha1"},"location":{"spanId":999}}]}'
      })
    })
```